ProcessMaker/processmaker
# SPIKE- Review conclusions and work on potential fixes

## Issue & Reproduction Steps
There is an error when performing an upgrade in Processmaker.
In the Cloudops team steps the Lang directory is not copied, because when the user makes their own updates to their language files, these changes are lost with the default tags that are in place.

## Solution
- A change is made in the Cloudops team steps and the Lang directory is copied but with the name Lang.orig.
- An artisan command has also been created to synchronize those two directories and maintain the user's changes.

## How to Test
- make changes to the tags using the `/admin/translations` interface (saving the changes creates a lang.orig directory).
- add new tags to the files in the 'lang.orig' directory (simulating that new tags were added by the ProcessMaker upgrade).
- run the command `php artisan processmaker:sync-translations`
- in the lang folder all the tags should be synchronized without losing the changes made by the user.

## Related Tickets & Packages
- [FOUR-17777](https://processmaker.atlassian.net/browse/FOUR-17777)
- [FOUR-17479](https://processmaker.atlassian.net/browse/FOUR-17479)


ci:next

## Code Review Checklist


[FOUR-17777]: https://processmaker.atlassian.net/browse/FOUR-17777?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
